src/perf_context.rs

```diff
@@ -389,4 +447,26 @@ mod test {
         assert_eq!(ctx.internal_delete_skipped_count(), n / 2);
         assert_ne!(ctx.seek_internal_seek_time(), 0);
     }
+
+    #[test]
+    fn test_iostats_context() {
+        let temp_dir = TempDir::new("test_iostats_context").unwrap();
+        let mut opts = DBOptions::new();
+        opts.create_if_missing(true);
+        let db = DB::open(opts, temp_dir.path().to_str().unwrap()).unwrap();
+
+        set_perf_level(PerfLevel::EnableTime);
+        let mut ctx = IOStatsContext::get();
+        assert_eq!(ctx.write_nanos(), 0);
```

Comment:
Nice work, could you add add test case or assert condition for all the other API? such as `ctx.bytes_written()`

Reply:
Done. It's not easy to simulate the `range_sync_nanos` field, so I just skip it here.